modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc

```diff
@@ -23,17 +11,15 @@ data:
 
 . From the {ocp-short} web console, select the *ConfigMaps* tab.
```

Comment:
Sort of out of scope for this PR, but when mentioning the OCP web console, I'd recommend saying whether it's in the Developer or Administrator perspective, and also listing admin permissions as a prereq if they're required (usually they are to edit / modify configmaps I think)

Reply:
That sounds good. I'm working with a SME on a separate PR to improve/reorg all of the _custom app config file_ content and update all the prereqs, etc. Will incorporate this feedback into those changes.